config/hello_client-proxy.json

```diff
@@ -18,7 +18,10 @@
   "applications": [
     {
       "name": "hello_client",
-      "id": "0x3456"
+      "id": "0x3456",
+      "max_dispatchers": "10",
+      "max_dispatch_time": "1000",
+      "threads": "2"
     }
   ],
   "clients": [
```

Comment:
I suggest to update the value of service id to be aligned as per COVESA protos

https://github.com/COVESA/uservices/blob/main/src/main/proto/example/hello_world/v1/hello_world_service.proto

Reply:
This is client id

Reply:
There was no change in this PR regarding service_id, hence I could not comment on the specific line. Apologies for that. 

I meant Service part in the config file 
 
"services": [
    {
        **"service": "0x3E7",**
        "instance": "0x0000",
        "unreliable": "30522"
    }